ddtrace/internal/ci_visibility/writer.py

```diff
@@ -87,6 +86,7 @@ def __init__(
         reuse_connections=None,  # type: Optional[bool]
         headers=None,  # type: Optional[Dict[str, str]]
         use_evp=False,  # type: bool
+        code_coverage=False,  # type: bool
```

Comment:
nit/preference: I think I'd maintain this as something that feels bool (eg: like `coverage_enabled`)